### ISSUE TITLE
Expose configuration for local JWT validation

### ISSUE THREAD

**Issue by robertjd** [stormpath/express-stormpath#168]
We want to allow the developer to specify local validation of JWTs, avoiding the rountrip to the API.  At the moment our cookie-based authentication and password-based oauth tokens will always do a remote stormpath validation, causing the network overhead with that request.

This depends on the resolution of https://github.com/stormpath/stormpath-framework-spec/pull/12, as that is where we are deciding the format of that configuration option:

**Comment by brentlemons** [comment]
Any luck on this issue?

**Comment by robertjd** [comment]
This is now available in the 2.3.1 release.  Please use this configuration to enable local validation:

```
{
  web: {
    oauth2: {
      password: {
        validationStrategy: 'local'
      }
    }
  }
}
```

More information can be found here: [Token Validation Strategy](http://docs.stormpath.com/nodejs/express/latest/authentication.html#token-validation-strategy)
